notepadqq.yaml

```diff
@@ -0,0 +1,35 @@
+name: notepadqq
+buildsystem: qmake
+config-opts:
+  - QMAKE_STRIP=
+  - INCLUDEPATH+="${FLATPAK_DEST}/include/QtWebKitWidgets"
+  - INCLUDEPATH+="${FLATPAK_DEST}/include/QtWebKit"
+  - LRELEASE=lrelease
+build-options:
+  env:
+    QMAKEPATH: "/app/lib"
+sources:
+  # Source0
+  - type: archive
+    url: https://github.com/notepadqq/notepadqq/archive/v1.4.8.tar.gz
+    sha256: 13fba9abd84c59de27fbe92f74e2763b57588fcf9c88af10ec67313b0abbc9d0
+  # Patch0
+  - type: patch
+    path: notepadqq-1.4.8-appdata.patch
+  # fix desktop file
+  - type: shell
+    commands:
+      - echo 'fix desktop file';
+        desktop-file-edit --remove-key=OnlyShowIn "support_files/shortcuts/notepadqq.desktop";
+post-install:
+  # create symlink
```

Comment:
Really not needed, same for the `echo`. The commands are printed, what they do is obvious.

Reply:
> Really not needed, same for the `echo`. The commands are printed, what they do is obvious.

Not everything is printed to stdout/stderr and such messages let you know where you are when something went wrong.
Moreover, not everything is so obvious and I try to keep the same style in every place.